File: src/main/java/it/posteitaliane/omp/UI/ApplicationServlet.java
package it.posteitaliane.omp.UI;

import com.vaadin.annotations.VaadinServletConfiguration;
import com.vaadin.server.VaadinServlet;

import javax.servlet.annotation.WebServlet;

@WebServlet(value = "/*", asyncSupported = true)
@VaadinServletConfiguration(productionMode = false, ui = it.posteitaliane.omp.scalaUI.Application.class, widgetset = "it.posteitaliane.omp.UI.AppWidgetSet")
public class ApplicationServlet extends VaadinServlet {
}

File: src/main/scala/it/posteitaliane/omp/UI/ApplicationServlet.scala
package it.posteitaliane.omp.UI

import javax.servlet.annotation.WebServlet
import com.vaadin.annotations.VaadinServletConfiguration
import com.vaadin.server.VaadinServlet

@WebServlet(value = Array("/*"), asyncSupported = true)
@VaadinServletConfiguration(productionMode = false, ui = classOf[Application], widgetset = "it.posteitaliane.omp.UI.AppWidgetSet")
class ApplicationServlet extends VaadinServlet